### PR TITLE
Add stage instance start notification, guild animated banner, application command locale, student hub

### LINKED ISSUE #737: Event::READY not triggering
Bot runs flawlessly, events like `GUILD_CREATE`, `GUILD_MEMBER_ADD` all trigger. But for some reason `READY` never does. Possible bug?

**Environment**
- PHP Version:
    - 8.0.x
- DiscordPHP Version:
    - 7.0.2

### PR DESCRIPTION
**Pull request by SQKo**
Bump version to 7.1.0, please merge and release #773 first before this.
- Add `StageInstance::send_start_notification` and `mention_everyone` permission for Stage Channels https://github.com/discord/discord-api-docs/pull/4706
- Add `Guild::feature_animated_banner` https://github.com/discord/discord-api-docs/pull/3948
- ~~Add `Event::READY` handler~~

### CODE REVIEW

**Comment by SQKo** [comment]
Also added Application Command locale https://github.com/discord/discord-api-docs/pull/4653, not tested yet but should be fine to be merged.

**Comment by SQKo** [comment]
Also add Student Hub type https://github.com/discord/discord-api-docs/pull/4701

File: src/Discord/Parts/Guild/Guild.php [hunk 1]
```diff
@@ -605,6 +635,72 @@ public function createEmoji(array $options, ?string $filepath = null, ?string $r
...
+
+        $contents = file_get_contents($filepath);
+
+        // TODO fix structure
+        $multipart = new Multipart([
+            'name' => $options['name'],
+            'description' => $options['description'],
+            'tags' => $options['tags'],
+            'file' => $contents,
+        ]);
```
Comment (davidcole1340) [inline src/Discord/Parts/Guild/Guild.php:688]:
```suggestion
        $multipart = new Multipart([
            'name' => $options['name'],
            'description' => $options['description'],
            'tags' => $options['tags'],
            'file' => $contents,
        ]);
        $multipart = new Multipart([
            [
                'name' => 'name',
                'content' => $options['name'],
            ],
            [
                'name' => 'description',
                'content' => $options['description'],
            ],
            [
                'name' => 'tags',
                'content' => $options['tags'],
            ],
            [
                'name' => 'file',
                'filename' => basename($filepath),
                'content' => $contents,
            ],
        ]);
```

**Comment by SQKo** [comment]
Added createSticker() with some todo:
https://github.com/discord-php/DiscordPHP/projects/5#card-79941275

**Review by davidcole1340 — APPROVED** [review]

**Pull request merged**